navigation/presentation.html

```diff
@@ -2,43 +2,1222 @@
+              <p><b>React-native-router-flux: </b>based on React Navigation</p>
+              <p><b>React Router Native: </b>easy to use if familiar with ReactJS and react-router-dom</p>
+            </div>
+            <div class="logos">
+              <img src="assets/images/logo_react_native_navigation.png" alt="logo" />
+              <img src="assets/images/logo_react_navigation.svg" alt="logo" />
+              <img src="assets/images/logo_react_router_native.svg" alt="logo" />
+            </div>
+          </div>
+
+          <aside class="notes">
+            In React Native, we have the option of setting up navigation by creating our own native modules or by building our own JavaScript-based implementation.
+            There are also a number of community-supported libraries that we can use for either of these approaches.
+            <br /><br />
+            Some of them you can see on the slide.
+            <br />
+            - <strong>React navigation </strong>is widely used, well documented. It delivers out-of-the-box, pre-styling navigators such as Stack Navigator, Bottom Tab… You can also customize or extend any part of React Navigation to fit your needs.
+            <br />
+            - <strong>React Native Navigation </strong>provided by Wix engineering. This is a popular alternative to React Navigation and may be a better fit for you if you are trying to integrate React Native into an existing large native app. This library doesn’t work with Expo (expo is a framework to build React Native apps, it provides you with a list of tools that simplify the creation and testing of RN app).
+            <br />
+            - <strong>react-native-router-flux </strong>- this library is based on React Navigation but provides you with a different API to interact with it.
+            <br />
+            - <strong>React Router Native </strong>- this library is super easy to use if you are a web developer or an ex web developer and familiar with both ReactJS and react-router-dom. Includes everything from react-router and adds a few APIs that are specific to React Native, including &lt;NativeRouter&gt; and a native version of &lt;Link&gt;
+            <br /><br />
+            <strong>The most popular and the most frequently used on real projects are the first two.
+              They are also recommended by react-native team.
+            </strong>
+          </aside>
+        </section>
+        <!-- 5 slide -->
+        <section class="centred-column-text-slide">
+          <h4 class="slide-name">React navigation</h4>
+          <div>
+            <p>routing and navigation for Expo and React Native apps</p>
+            <p>completely customizable</p>
+            <p>components built for iOS and Android</p>
+            <p>hooks</p>
+            <p>docs <a href="https://reactnavigation.org/" target="_blank">https://reactnavigation.org/</a></p>
+          </div>
```

Comment:
this slide dramatically needs a React Navigation logo to the left or above the list of features

Reply:
added